grails-app/services/org/pih/warehouse/inventory/StockMovementService.groovy

```diff
@@ -2578,6 +2578,13 @@ class StockMovementService {
                             stepNumber  : null,
                             uri         : g.createLink(controller: 'goodsReceiptNote', action: "print", id: stockMovement?.shipment?.id, absolute: true),
                             hidden      : !stockMovement?.shipment?.receipt
+                    ],
```

Comment:
Move this before the document with the `stepNumber: null`

Reply:
ok